src/init/features/project.ts

```diff
@@ -6,14 +6,16 @@ import * as FirebaseError from "../../error";
 import {
   FirebaseProjectMetadata,
   getFirebaseProject,
-  listFirebaseProjects,
+  getProjectPage,
 } from "../../management/projects";
 import * as logger from "../../logger";
 import { promptOnce, Question } from "../../prompt";
 import * as utils from "../../utils";
 
-const NO_PROJECT = "[don't setup a default project]";
-const NEW_PROJECT = "[create a new project]";
+const MAXIMUM_PROMPT_LIST = 100;
+const NO_PROJECT = "Don't setup a default project";
```

Comment:
nit: `set up` (verb) should be used here, rather than `setup` (noun)

Reply:
Done